### ISSUE TITLE
SimCity 2000 Special Edition sound card initialisation pops regression (dev)

### ISSUE THREAD

**Issue by Grounded0** [dosbox-staging/dosbox-staging#2814]
### Are you using the latest Dosbox-Staging Version?

- [X] I have checked releases and am using the latest release.

### Different version than latest?

Version 0.81.0-alpha (v0.81.0-alpha-1558-g55969d)

### What Operating System are you using?

Other

### If Other OS, please describe

macOS 12.6.5/x86-64

### Relevant hardware info

2017 15" MacBook Pro

### Have you checked that no other similar issue already exists?

- [X] I have searched and not found similar issues.

### A clear and concise description of what the bug is.

There's sound card initialisation pops in SimCity 2000 Special Edition with development builds. It's a regression against 0.80.1.

Audio recording:

https://github.com/dosbox-staging/dosbox-staging/assets/48192273/23537d5c-9641-4db3-b009-887f5581b0c4

This is what runs it (`SC2000SE.BAT`):

```
imgmount -u d
imgmount d y:\sc2000se.iso -t cdrom
@echo off
rem Batch file for SimCity 2000 DOS Special Edition
rem
rem Facilitates launching of WillTV from within SC2K.
rem
rem by Kevin O'Hare 2/16/96
rem    Last Updated  3/6/96
rem
rem Copyright Maxis 1996
rem

INTRO

:RunSC2K
SC2K %1
if errorlevel 55 goto RunWillTV
goto END

:RunWillTV
WILLTV
goto :RunSC2K

:END
```

The pops are audible if you run this batch file or you separately execute `SC2K %1`.

### Steps to reproduce the behaviour.

Explain how to reproduce
1. Run SimCity 2000 Special Edition with development builds
2. Hear pops
3. Profit

### Your configuration

I haven't rebased my dosbox conf file for 0.81 dev.

```ini
[sblaster]
sbtype              = sb16
sbbase              = 220
irq                 = 7
dma                 = 1
hdma                = 5
sbmixer             = true
sbwarmup            = 100
oplmode             = auto
sb_filter           = modern
sb_filter_always_on = true
opl_filter          = off
cms_filter          = on
```

### Code of Conduct & Contributing Guidelines

- [X] Yes, I agree.

**Comment by johnnovak** [comment]
You flagged it as regression @Grounded0 so can you include info about the release / fork where the pop does not happen?

**Comment by Grounded0** [comment]
0.80.1.

**Comment by Grounded0** [comment]
Added audio recording of the issue.

**Comment by kcgen** [comment]
Great catch, @Grounded0 !

Copy and paste typo on my part, fixed in: https://github.com/dosbox-staging/dosbox-staging/commit/c156b0fe426ccc4d80ad4876becc217f2b55f1cd

**Comment by johnnovak** [comment]
> Great catch, @Grounded0 !
> 
> Copy and paste typo on my part, fixed in: [c156b0f](https://github.com/dosbox-staging/dosbox-staging/commit/c156b0fe426ccc4d80ad4876becc217f2b55f1cd).

Nice on @kcgen !
